Sources/BrowserServicesKit/PrivacyConfig/AppPrivacyConfiguration.swift

```diff
@@ -142,6 +142,11 @@ public struct AppPrivacyConfiguration: PrivacyConfiguration {
     public func isSubfeatureEnabled(_ subfeature: any PrivacySubfeature,
                                     versionProvider: AppVersionProvider,
                                     randomizer: (Range<Double>) -> Double) -> Bool {
+        // TODO: Do not ship this - temporary, only for testing
+        if let networkProtectionSubfeature = subfeature as? NetworkProtectionSubfeature, networkProtectionSubfeature == NetworkProtectionSubfeature.waitlist {
+            return true
+        }
```

Comment:
The waitlist feature flag doesn't exist in the remote config yet (it won't until after this ships), so the feature has been hardcoded on.

I plan to replace this with a hardcoded privacy config URL which _does_ contain the flag with a 50% rollout value, so that it can be tested most accurately.